pipsi/__init__.py

```diff
@@ -270,7 +285,11 @@ def _cleanup():
             return False
 
         # Install virtualenv, use the pipsi used python version by default
-        args = [sys.executable, '-m', 'virtualenv', '-p', python or sys.executable, venv_path]
+        if venv_pkg == 'venv':
```

Comment:
this has to check the python versions of the desired python, if python2 is asked for, there wont be a venv even if the installing python will have it